From 3ed25d2b2b5ffa699e4a3f38568e3ee6a7984624 Mon Sep 17 00:00:00 2001
From: Pramod Kumar <pramod.kumar_1@nxp.com>
Date: Wed, 8 May 2019 17:55:24 +0530
Subject: [PATCH 143/235] dt-bindings: arm: nxp: Add device tree binding for
 ls1046a-frwy board

Add "fsl,ls1046a-frwy" bindings for ls1046afrwy board based on ls1046a SoC

Signed-off-by: Vabhav Sharma <vabhav.sharma@nxp.com>
Signed-off-by: Pramod Kumar <pramod.kumar_1@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 Documentation/devicetree/bindings/arm/fsl.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/fsl.txt b/Documentation/devicetree/bindings/arm/fsl.txt
index 9eef2c22e359..e54d272a248f 100644
--- a/Documentation/devicetree/bindings/arm/fsl.txt
+++ b/Documentation/devicetree/bindings/arm/fsl.txt
@@ -183,6 +183,10 @@ LS1046A ARMv8 based RDB Board
 Required root node properties:
     - compatible = "fsl,ls1046a-rdb", "fsl,ls1046a";
 
+LS1046A ARMv8 based FRWY Board
+Required root node properties:
+    - compatible = "fsl,ls1046a-frwy", "fsl,ls1046a";
+
 LS1088A SoC
 Required root node properties:
     - compatible = "fsl,ls1088a";
-- 
2.17.1

